examples/v1beta1/general-purpose.yaml

```diff
@@ -0,0 +1,33 @@
+# This example NodePool will provision general purpose instances
```

Comment:
(nit) When a YAML file has a `---` line, I put one at the start as well.

That helps people learn that `---` is a document start marker, not a separator.